Documentation/Signoffs/launcher-aiming-signoff.md

```diff
@@ -0,0 +1,59 @@
+# Launcher Aiming
+## Functionality
+The Launcher Aiming subsystem will take in signals from the sensor post reciever and head on projectile sensor, determine the position of the target and adjust the launcher's position accordingly. This subsystem will also calculate the timing to fire the projectile and send a signal to the launcher firing subsystem. The launcher's position is controlled by two motors: one to adjust the yaw (turntable) and one to adjust the pitch (launch angle).
+## Constraints 
+* C1: Create an interceptor capable of functioning
+on its own without outside interaction.
+* C3: Create an interceptor capable of fitting inside

```

Comment:
Same question as above. The constraint doesn't apply here. It applies to the total system.